src/components/util/RivetIcons.jsx

```diff
@@ -37,6 +38,15 @@ const IconCore = ({ children, ...attrs }) => (
   </svg>
 );
 
+const icoBell = (attrs) => (
```

Comment:
I think this is a typo. It should be `iconBell`, not `icoBell`.

Reply:
Yeah, I agree. The rest of the icons are labeled as `ico*`, though. I think we can change them in another PR.